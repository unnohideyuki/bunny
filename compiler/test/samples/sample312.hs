main = do print $ [LT ..]
          print $ [GT, EQ ..]
