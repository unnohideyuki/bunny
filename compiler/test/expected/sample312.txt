[LT,EQ,GT]
[GT,EQ,LT]
